packages/dd-trace/src/ci-visibility/intelligent-test-runner/get-itr-configuration.js

```diff
@@ -0,0 +1,88 @@
+const request = require('../../exporters/common/request')
+const id = require('../../id')
+
+function getItrConfiguration ({
+  site,
+  env,
+  service,
+  repositoryUrl,
+  sha,
+  osVersion,
+  osPlatform,
+  osArchitecture,
+  runtimeName,
+  runtimeVersion,
+  branch
+}, done) {
+  const url = new URL(`https://api.${site}`)
+
+  const apiKey = process.env.DATADOG_API_KEY || process.env.DD_API_KEY
+  const appKey = process.env.DATADOG_APP_KEY ||
+    process.env.DD_APP_KEY ||
+    process.env.DATADOG_APPLICATION_KEY ||
+    process.env.DD_APPLICATION_KEY
+
+  if (!apiKey || !appKey) {
+    done(new Error('App key or API key undefined'))
+    return
+  }
+
+  const options = {
+    path: '/api/v2/libraries/tests/services/setting',
+    method: 'POST',
+    headers: {
+      'dd-api-key': apiKey,
+      'dd-application-key': appKey,
+      'Content-Type': 'application/json'
+    },
+    timeout: 15000,
```

Comment:
This is way too high. I also didn't notice that there are similar high timeouts elsewhere. Combined, these could be blocking the test suite for over a minute if there is a problem with the backend.

Reply:
yeah I'll reduce this one. For the skippable suites call: I don't think we can reduce the timeout yet, the operation the backend needs to do is quite heavy (performance is being worked on). The majority of the times we'll make it up by skipping a lot 😄